modules/compiler/test/lit/passes/work-item-loop-remove-replaced.ll

```diff
@@ -0,0 +1,31 @@
+; Copyright (C) Codeplay Software Limited
+;
+; Licensed under the Apache License, Version 2.0 (the "License") with LLVM
+; Exceptions; you may not use this file except in compliance with the License.
+; You may obtain a copy of the License at
+;
+;     https://github.com/codeplaysoftware/oneapi-construction-kit/blob/main/LICENSE.txt
+;
+; Unless required by applicable law or agreed to in writing, software
+; distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+; WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+; License for the specific language governing permissions and limitations
+; under the License.
+;
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
We're not 100% consistent across files but we generally have a blank line after the `SPDX-License-Identifier`, and another one after `RUN:`. (And I'm not sure if the double blank lines later are intentional.)

Reply:
Should be fixed now, thanks